api/model-server.yaml

```diff
@@ -314,6 +314,18 @@ paths:
           $ref: '#/components/responses/500'
         "200":
           $ref: '#/components/responses/versionDelta'
+  /v2/repositories/{repository}/delete:
+    post:
+      operationId: deleteRepository
+      parameters:
+        - name: repository
+          in: "path"
+          required: true
+          schema:
+            type: string
+      responses:
```

Comment:
Which response is given when a repository is to be deleted that does not exist? Should be specified here.